### DISCUSSION TITLE
Important Announcement - Development Update

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#531] [Announcements]
Hey,

This is a sad news, but **_I am going to slowly quit Eden's development_.**

Eden is almost 1 year old. The plugin somehow creates a small community. I am very happy about this.

But, from nothing to here, the plugin is now getting bigger and bigger. I've been receiving a lot of questions, bug reports, and feature requests. Some of the feature requests is already half a year old already. Stuff like these are impacting my daily life. As I said in built-by-bit (The only platform I've advertised), the reason why I open source the plugin is because I don't have time to maintain. But I still tried to spent time to continue the development and keep the plugin bug-free and feature-rich, and because of this, it makes me cannot do stuff I enjoy.

For those who know me previously, I am a content creator. But when I try to spent time into Eden's development, it also consume the time I saved for recording and editing. From video per 3 days, to video per month, and now I haven't posted a video for 2 months already. This is slowly killing my channel and the algorithm.

Currently the latest Minecraft version is 1.20. Eden is a plugin developed for 1.8, which released 11 years ago. As a developer who already quitted competitive Minecraft PvP, there's no point to stay in 1.8.

Thank for reading this, and thank you for the opportunity. If you have more questions, you are welcome to dm me via email, or leave a comment.

<h2>Will I still receive support?</h2>
Yes. I will still giving support until September. But feature request will be disabled.

<h2>Will there be someone who continue to development?</h2>
No. If you are interested in continue to work on the plugin, you are welcome to fork the plugin and do whatever you want. Just be sure to give credit.

<h2>What happens after September?</h2>
The github repo will become archived. You will still able to view the source code.

**Comment by lulu2002** [comment]
You've already done your best! there's no reason being sad :)

**Comment by HierarchyMan** [comment]
This is sad news but Eden is already a gift to this small community
Good luck

**Comment by Chromaah** [comment]
Thank you @GoodestEnglish for being such a nice and respectful developer, and having patience for all the people that needed help with the plugin. I appreciate your hard work and dedication that was put into this project, and hope to see you with some other projects sometime in the future. (The fact that it was all free was so shocking too.) This plugin is such a great alternative to the expensive plugins like frost and honestly very surprised that you made it this far. This plugin is really well made and you kept constantly updating it when possible which really shows all the things that are possible in the development community.

Overall thank you for all that you have done, we respect your decision and hope you continue doing what you love, whether that is developing private plugins, creating content, or anything else.

**Comment by minehostil** [comment]
thanks for all your time and patience, you are undoubtedly a great developer, thanks for leaving us this incredible plugin, good luck and take your time with things

**Comment by serumanooo** [comment]
Thanks for all the support you gave to the Eden community, Eden is by far the best practice plugin ive ever seen and you are a really good developer and did a really great job on it. Also thanks for all the patience you had with the bug reports and feature requests.

**Comment by Hwaanx** [comment]
Very good plugin i achieved my dreams with this plugin

**Comment by ushereats** [comment]
I'm a condom dealer
